docs/anchoring-assets/overview.mdx

```diff
@@ -0,0 +1,30 @@
+
+Stellar has anchor services operating worldwide. View the [Anchor Directory](https://resources.stellar.org/anchors?) for more information on existing Stellar anchors.
+
+Anchors can issue their own assets on the Stellar network, or they can honor assets that already exist. To learn about issuing assets, check out the [Issue Assets section](https://developers.stellar.org/docs/category/issue-assets).
+
+This documentation will instruct you on how to become an anchor. To understand how to integrate anchor services into your blockchain-based application, check out the [Build Apps section](https://developers.stellar.org/docs/category/deposit-and-withdraw-from-anchors). If you’re looking specifically for MoneyGram Access, see the [Integrate with MoneyGram Access tutorial](https://developers.stellar.org/docs/tutorials/moneygram-access-integration-guide).
+
+## Stellar Ecosystem Proposals (SEPs) and interoperability
+
+Stellar is an open-source network that is designed to interoperate with traditional financial institutions, various types of assets, and other networks. Network participants implement Stellar Ecosystem Proposals (SEPs) to ensure they can interoperate with other products and services on the network. SEPs are publicly created, open-source documents that live in a [GitHub repository](https://github.com/stellar/stellar-protocol/tree/master/ecosystem#stellar-ecosystem-proposals-seps) and they define how asset issuers, applications, exchanges, and other service providers should interact and interoperate.
+
+Read more about SEPs in the [SEPs section](https://developers.stellar.org/docs/fundamentals-and-concepts/stellar-ecosystem-proposals).
+
+As an anchor, the most important SEPs are [SEP-0024: Hosted Deposit and Withdrawal](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0024.md), and [SEP-0031: Cross Border Payments API](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0031.md#payment-flow). You’ll also work with [SEP-0010: Stellar Authentication](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0010.md), [SEP-0012: KYC API](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0012.md), and [SEP-0038: Anchor RFQ API](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0038.md).
+
+## Anchor Platform versus setting up an anchor from scratch
+
+The anchor documentation will help you set up an anchor using the Anchor Platform or from scratch.
+
+The Anchor Platform is the easiest and fastest way to deploy anchor services compatible with SEPs. **You must use the Anchor Platform if you’re setting up an anchor to support SEP-0024: Hosted Deposits and Withdrawal**. The Anchor Platform does not yet have SEP-0031 capabilities but will in the future.
```

Comment:
They don't *have to* use the Anchor Platform but we strongly encourage it, and SEP-31 is actually available in the Anchor Platform today, we just haven't invested in documenting it yet.

Reply:
I'll change the wording. We're deprecating the From Scratch sections though, right? Do we have plans to write those again at some point? Otherwise we'd say something like: "We strongly encourage using the Anchor Platform to set up an anchor. If you want to set up an anchor from scratch, we can't help you." Not really like that, but you get the idea.

Reply:
I think it's better to list benefits of using AP (official support, new features, easy to integrate with, etc)

Reply:
Lol yeah I wouldn't actually say that. I'll rewrite this tomorrow.